pgx.opam

```diff
@@ -20,7 +20,7 @@ depends: [
   "sexplib0" {>= "v0.11.0"}
 
   "bisect_ppx" {build & >= "1.4.1"}
-  "dune" {build & >= "1.1.0"}
+  "dune" {>= "1.1.0" & < "2.0"}
```

Comment:
What's wrong with dune >= 2.0?

Reply:
There was a problem with inline_tests on core versions 0.11. This won't be required if its fine to just depend on versions of core that use dune instead of jbuilder